### DISCUSSION TITLE
Remove nightcore hats from maps without appropriate slider tick rate

### DISCUSSION THREAD

**Discussion by cl8n** [ppy/osu#29671] [Ideas]
i.e. do what stable does:
> `nightcore-hat.wav` is played on every odd [quaver](https://en.wikipedia.org/wiki/Eighth_note), but only if the [slider tick rate](https://osu.ppy.sh/wiki/en/Beatmapping/Slider_tick_rate) of the beatmap is a multiple of 2

not that parity with stable really matters for such a niche feature like this, but I think this helped nightcore sound less obnoxious on songs that clash hard with a constant eighth note rhythm going on in the background.

**Comment by peppy** [comment]
@cl8n do you have a beatmap in mind which can be used to test this? https://github.com/ppy/osu/compare/master...peppy:osu:nightcore-limited-hats?expand=1

**Reply by daifkyou** [reply]
I just noticed this discussion which appears to be a duplicate of #13513. In [my comment](https://github.com/ppy/osu/issues/13513#issuecomment-3539141199) I give [this example](https://osu.ppy.sh/s/317538) (tick rate 3) in a compound metre (12/8) which is greatly affected because each beat was intended to be subdivided into triplets. The person who raised the issue also posted videos of what appears to be [this map](https://osu.ppy.sh/s/1148220) (tick rate 1), which is in 4/4 but where the hats were just clearly not meant to be. For regression, [these](https://osu.ppy.sh/s/1179980#osu/2742359) [difficulties](https://osu.ppy.sh/s/425169#osu/951132) (tick rate 2) should have hats. Unfortunately I'm not an old enough player to casually have tick rate 4 maps lying around.
